Pages/Shared/_Layout.cshtml.css

```diff
@@ -39,10 +39,9 @@ button.accept-policy {
   line-height: inherit;
 }
 
-.footer {
-  position: absolute;
-  bottom: 0;
-  width: 100%;
-  white-space: nowrap;
-  line-height: 60px;
-}
+@media (max-width: 768px) {
+    .site-footer {
+        line-height: 40px;
```

Comment:
Add responsive rules for .site-footer to prevent layout breaking on smaller screens: